Primera Iteración (diseño lógico): se mapea el diagrama ER

AndresFSH opened the pull request:
Se convirtieron las entidades en las tablas correspondientes, teniendo en cuenta que:
- Cada entidad representa una tabla, y cada atributo una columna.
- Una relación N:M genera una nueva tabla intermedia con relación uno a muchos.
- Se deben mantener las entidades y atributos existentes en el ER.

AndresFSH committed:
Primera Iteración (diseño lógico): se mapea el diagrama ER

Se convirtieron las entidades en las tablas correspondientes,
teniendo en cuenta que:
- Cada entidad representa una tabla, y cada atributo una columna.
- Una relación N:M genera una nueva tabla intermedia con relación uno a muchos.
- Se deben mantener las entidades y atributos existentes en el ER.

sandredev reviewed (changes requested):
El archivo que agregas como diagrama en la carpeta "diseño lógico/" tiene un nombre no adecuado para la estructura del proyecto. Con GitHub ya estamos manejando versiones, "primeraversión" no es un nombre adecuado para un archivo del proyecto. 

Cambia el nombre y vuelve a hacer pull request.

AndresFSH committed:
Primera Iteración( diseño lógico ): corrección de errores

Se presentaban errores que fueron corregidos.

 - El nombre del diagrama relacional cambió de 'primeraiteracion.png' a 'imagenDiseñoLogico.png'.
 - Se eliminó el modelo de ejemplo.
